testament/important_packages.nim

```diff
@@ -74,7 +74,7 @@ pkg "gnuplot", "nim c gnuplot.nim"
 pkg "hts", "nim c -o:htss src/hts.nim"
 pkg "httpauth"
 pkg "illwill", "nimble examples"
-pkg "inim"
+#pkg "inim"
```

Comment:
```suggestion
pkg "inim", allowFailure = true
```

Reply:
What's the point of `allowFailure`? It's a mess --- unreadable logs, you don't know which package *really* failed, you don't know when it starts to work again and then when it starts to fail again. The current state of failing projects should be reflected in important_packages.nim, not hidden in some enormous CI log.

Reply:
With #17757 they would be all in one batch so wouldn't spam the other logs; and eventually we can have the CI notify us when a package starts to work again.

Reply:
Did it this way anyway.